src/System.Security.Cryptography.Algorithms/tests/RandomNumberGeneratorTests.netcoreapp.cs

```diff
@@ -279,7 +279,8 @@ private static void VerifyDistribution(ReadOnlySpan<int> numbers, double expecte
             foreach ((_, int occurences) in observedNumbers)
             {
                 double percentage = occurences / (double)numbers.Length;
-                Assert.True(Math.Abs(expected - percentage) < tolerance, "Occurred number of times within threshold.");
+                double actual = Math.Abs(expected - percentage);
+                Assert.True(actual < tolerance, $"Occurred number of times within threshold. Actual: {actual}");
```

Comment:
You could consider Assert.InRange.  But this is also fine.

Reply:
I thought about it (no idea why I didn't do that in the first place) but I noticed that `InRange` is inclusive in its upper bound which is a slight change in behavior from the original assertion. It's probably fine, but I opted for the change that required me to think the least.